mne/label.py

```diff
@@ -1659,6 +1660,159 @@ def _grow_nonoverlapping_labels(subject, seeds_, extents_, hemis, vertices_,
     return labels
 
 
+def random_parcellation(subject, n_parcel, hemis, subjects_dir=None,
+                        surface='white'):
+    """Generate random cortex parcellation by growing labels.
+
+    This function generates a number of labels which don't intersect and
+    cover the whole surface. Regions are growing around randomly chosen
+    seeds.
+
+    Parameters
+    ----------
+    subject : string
+        Name of the subject as in SUBJECTS_DIR.
+    n_parcel : int
+        Total number of cortical parcels
+    hemis : array | int
+        Hemispheres to use for the labels (0: left, 1: right).
+    subjects_dir : string
+        Path to SUBJECTS_DIR if not set in the environment.
+    surface : string
+        The surface used to grow the labels, defaults to the white surface.
+
+    Returns
+    -------
+    labels : list of Label
+        Random cortex parcellation
+    """
+    subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+    hemis = np.atleast_1d(hemis)
+    hemis = np.array(['lh' if h == 0 else 'rh' for h in hemis])
+
+    # load the surfaces and create the distance graphs
+    tris, vert, dist = {}, {}, {}
+    for hemi in set(hemis):
+        surf_fname = op.join(subjects_dir, subject, 'surf', hemi + '.' +
+                             surface)
+        vert[hemi], tris[hemi] = read_surface(surf_fname)
+        dist[hemi] = mesh_dist(tris[hemi], vert[hemi])
+
+    # create the patches
+    labels = _cortex_parcellation(subject, n_parcel, hemis, vert, dist)
+
+    # add a unique color to each label
+    colors = _n_colors(len(labels))
+    for label, color in zip(labels, colors):
+        label.color = color
+
+    return labels
+
+
+def _cortex_parcellation(subject, n_parcel, hemis, vertices_, graphs):
+    """Random cortex parcellation."""
+    labels = []
+    parcel_size = np.floor((len(vertices_[hemis[0]]) +
+                            len(vertices_[hemis[1]])) / n_parcel)
```

Comment:
I think

`len(vertices_[hemis[0]]) + len(vertices_[hemis[1]])) // n_parcel`

is nicer and is the same